code_challenges.rb

```diff
@@ -34,15 +46,55 @@
 # Expected output: 'bby Rd'
 
 
+def remove_vowels string
+  string.delete 'aeiouAEIOU'
+end
 
+p remove_vowels album1
+p remove_vowels album2
+p remove_vowels album3
 
 
 # --------------------4) Create a class called Bike that is initialized with a model, wheels, and current_speed. The default number of wheels is 2. The current_speed should start at 0. Create a get_info method that returns a sentence with all the data from the bike object.
 
 # Expected output example: 'The Trek bike has 2 wheels and is going 0 mph.'
 
-
-
+class Bike
+  attr_accessor :model, :wheels, :current_speed
+  def initialize model
+    @model = model
+    @wheels = 2
+    @current_speed = 0
+  end
+
+  def get_info
+    # puts "The #{model} bike has #{wheels} wheels and is going #{current_speed} mph."
+    puts "The #@model bike has #@wheels wheels and is going #@current_speed mph."
+  end
+
+  def pedal_faster
+    @current_speed += 10
+  end
+
+  def brake
+    (@current_speed > 15) ? @current_speed -= 15 : @current_speed -= @current_speed
```

Comment:
Just a formatting suggestion for longish ternary operations
```suggestion
    @current_speed > 15
      ? @current_speed -= 15 
      : @current_speed -= @current_speed
```